src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9094,7 +9094,7 @@ void func_808458D0(Player* this, PlayState* play) {
     }
 }
 
-s32 func_80845964(PlayState* play, Player* this, CsCmdActorAction* arg2, f32 arg3, s16 arg4, s32 arg5) {
+s32 func_80845964(PlayState* play, Player* this, CsCmdActorCue* arg2, f32 arg3, s16 arg4, s32 arg5) {
```

Comment:
```suggestion
s32 func_80845964(PlayState* play, Player* this, CsCmdActorCue* cue, f32 arg3, s16 arg4, s32 arg5) {
```
Since you do this for all others